### ISSUE TITLE
include_weekday_button="0" still displaying weekday button

### ISSUE THREAD

**Issue by paulnagle** [radius314/crouton#45]
[bmlt_tabs include_weekday_button="0"]

The weekday button is still displayed

<img width="1622" alt="screenshot 2018-10-02 at 21 54 26" src="https://user-images.githubusercontent.com/3714999/46376381-c144cc80-c68d-11e8-929e-c62c0867e513.png">

**Comment by pjaudiomv** [comment]
I don't think this is an actual bug. include_weekday_button="0" will only work if you also do view_by="city"
since view by weekday is the default, you must do this.

for ex. include_city_button="0" and view_by="city"
will still show the city button however just doing include_city_button="0" will make the city button disappear because default view_by i weekday.

that being said maybe if oyu are doing  include_weekday_button="0" we should assume you also want to do view_by="city"

**Comment by radius314** [comment]
is this a regression?

**Comment by radius314** [comment]
ok, might be valid but won't be addressing this in 2.0.0
